src/integrationTest/groovy/wooga/gradle/appcenter/AppCenterPluginIntegrationSpec.groovy

```diff
@@ -100,6 +100,33 @@ class AppCenterPluginIntegrationSpec extends IntegrationSpec {
+        "publishEnabled"        | true                  | _                                        | 'yes'             | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | 'YES'             | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | false             | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | 0                 | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | "FALSE"           | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | 'n'               | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | 'no'              | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | 'NO'              | PropertyLocation.env
+        "publishEnabled"        | true                  | _                                        | true              | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 1                 | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | "TRUE"            | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'y'               | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'yes'             | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'YES'             | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | false             | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 0                 | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | "FALSE"           | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'n'               | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'no'              | PropertyLocation.property
+        "publishEnabled"        | true                  | _                                        | 'NO'              | PropertyLocation.property
```

Comment:
Are these values actually used? oO

Reply:
It just tests all different ways to provide a boolean value in string form.

Reply:
Yeah, I know, I was just impressed by "n"...

Reply:
I've seen it all. And my thinking. If I have to test for a String representation anyways and the effort to test other known variants is marginal, why not test it.

Reply:
I didn'te test the other kown values `True` and `False` which should also work by the way since I convert the value to downcase anyways.